.github/workflows/main.yml

```diff
@@ -75,6 +75,25 @@ jobs:
       with:
         go-version: ${{ matrix.go-version }}
 
+    - name: Test CRIT
+      run: |
+        make -C crit clean bin/crit
+
+  crit-windows:
+    runs-on: windows-latest
```

Comment:
We can matrix this to avoid duplication:

```
    runs-on: ${{ matrix.os }}
    strategy:
      matrix:
        os: [ubuntu-latest, windows-latest, macos-latest]
```

Reply:
In Linux, full testing is conducted, while on Windows and Mac, only build testing is performed. In my opinion, combining all of them may not be the best approach. Is it acceptable to proceed with testing by combining only Windows and Mac?